Refine `AssertionError` message for `NRowsMinGain` and `NRowsMaxGain` constraints

When the constraint `NRowsMinGain` is not met, the following exemplary `AssertionError` is thrown:

> AssertionError: TAB_NEW has 0.30231844350740233 of #rows of TAB_OLD. It was supposed to increase at least 0.5.

I find this message misleading as TAB_NEW has actually 130 % of #rows of TAB_OLD, or 30 % increase in #rows. 

As a minimal refinement I would suggest to replace the string "of #rows of" with "relative gain in #rows compared to" in the `assertion_text ` of `NRowsMinGain`.

This would change the message to
> AssertionError: TAB_NEW has 0.30231844350740233 relative gain in #rows compared to TAB_OLD. It was supposed to increase at least 0.5.

Additionally, one could express the numbers as percentage by multiplying `relative_gain` as well as `self.min_relative_gain` with 100, adding "%" to the string and omitting the "relative". That would give us:
> AssertionError: TAB_NEW has 30.231844350740233 % gain in #rows compared to TAB_OLD. It was supposed to increase at least 50 %.

Similarly, it would probably make sense to change the error message in `NRowsMaxGain`.

--- a/src/datajudge/constraints/nrows.py
+++ b/src/datajudge/constraints/nrows.py
@@ -115,9 +115,9 @@ class NRowsMaxGain(NRows):
             return True, "Row loss."
         relative_gain = (n_rows_factual - n_rows_target) / n_rows_target
         assertion_text = (
-            f"{self.ref.get_string()} has {relative_gain} relative gain in #rows compared to "
+            f"{self.ref.get_string()} has {relative_gain:%} gain in #rows compared to "
             f"{self.ref2.get_string()}. It was only allowed "
-            f"to increase {self.max_relative_gain}. "
+            f"to increase {self.max_relative_gain:%}. "
             f"{self.condition_string}"
         )
         result = relative_gain <= self.max_relative_gain
@@ -146,9 +146,9 @@ class NRowsMinGain(NRows):
             return False, "Row loss."
         relative_gain = (n_rows_factual - n_rows_target) / n_rows_target
         assertion_text = (
-            f"{self.ref.get_string()} has {relative_gain} relative gain in #rows compared to "
+            f"{self.ref.get_string()} has {relative_gain:%} gain in #rows compared to "
             f"{self.ref2.get_string()}. It was supposed "
-            f"to increase at least {self.min_relative_gain}. "
+            f"to increase at least {self.min_relative_gain:%}. "
             f"{self.condition_string}"
         )
         result = relative_gain >= self.min_relative_gain

--- a/src/datajudge/constraints/nrows.py
+++ b/src/datajudge/constraints/nrows.py
@@ -117,7 +117,7 @@ class NRowsMaxGain(NRows):
         assertion_text = (
             f"{self.ref.get_string()} has {relative_gain:%} gain in #rows compared to "
             f"{self.ref2.get_string()}. It was only allowed "
-            f"to increase {self.max_relative_gain:%}. "
+            f"to increase by {self.max_relative_gain:%}. "
             f"{self.condition_string}"
         )
         result = relative_gain <= self.max_relative_gain
@@ -148,7 +148,7 @@ class NRowsMinGain(NRows):
         assertion_text = (
             f"{self.ref.get_string()} has {relative_gain:%} gain in #rows compared to "
             f"{self.ref2.get_string()}. It was supposed "
-            f"to increase at least {self.min_relative_gain:%}. "
+            f"to increase at least by {self.min_relative_gain:%}. "
             f"{self.condition_string}"
         )
         result = relative_gain >= self.min_relative_gain

--- a/src/datajudge/constraints/nrows.py
+++ b/src/datajudge/constraints/nrows.py
@@ -115,7 +115,7 @@ class NRowsMaxGain(NRows):
             return True, "Row loss."
         relative_gain = (n_rows_factual - n_rows_target) / n_rows_target
         assertion_text = (
-            f"{self.ref.get_string()} has {relative_gain} of #rows of "
+            f"{self.ref.get_string()} has {relative_gain} relative gain in #rows compared to "
             f"{self.ref2.get_string()}. It was only allowed "
             f"to increase {self.max_relative_gain}. "
             f"{self.condition_string}"
@@ -146,7 +146,7 @@ class NRowsMinGain(NRows):
             return False, "Row loss."
         relative_gain = (n_rows_factual - n_rows_target) / n_rows_target
         assertion_text = (
-            f"{self.ref.get_string()} has {relative_gain} of #rows of "
+            f"{self.ref.get_string()} has {relative_gain} relative gain in #rows compared to "
             f"{self.ref2.get_string()}. It was supposed "
             f"to increase at least {self.min_relative_gain}. "
             f"{self.condition_string}"

